CIP-0119/README.md

```diff
@@ -188,6 +189,16 @@ It is also the case that CIP-100 allows for metadata to be saved within a govern
 
 Therefore it was decided to allow a provision for people to submit `imageObject`'s with a URL only if a hash was included OR with a base64 encoded image, and allow them to make the decision as to which was most appropriate for their use case.  
 
+#### Rationale for `doNotList`
+
+This field was intended to be used by DReps who wish to identify themselves via rich metadata but are not seeking to campaign for delegations.
+By not be listed via "DRep aggregation/campaign" tools the idea is that these DReps are less likely to attract unwanted delegation from ada holders.
+These DReps could be organizations want to use their ada to vote, in a transparent way on-chain but do not wish to vote on the behalf of others.
+
+It is expected that tooling such as block explorers will list DReps using `doNotList=true`. Whereas tooling built specifically for DRep campaign and delegation should respect the intent of this field. 
+
+This proposal cannot force tooling to respect this desire from DReps. DReps must be aware that any information anchored on-chain can be found via tooling, and may result in delegation.
```

Comment:
```suggestion
This proposal cannot force tooling to respect this desire from DReps. DReps must be aware that any information anchored on-chain can be found via tooling and may result in delegation.
```
(if we are fixing the other commas... this one may be wrong grammar but would be commonly accepted as OK)